rclpy/guard_conditions/examples_rclpy_guard_conditions/trigger_guard_condition.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+
+
+def main(args=None):
+
+    rclpy.init(args=args)
+    node = rclpy.create_node('snake_case')
```

Comment:
oops, I should have been more clear in my previous comment. I meant changing case style of the node name from CamelCase to [snake_case](https://en.wikipedia.org/wiki/Snake_case). I think the name used previously was nice and descriptive:
```suggestion
    node = rclpy.create_node('demo_guard_condition')
```

Reply:
Haha, I shouldn't have been so literal!